app/Http/Requests/UpdatePersonalInfoRequest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdatePersonalInfoRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return $this->user()->can('update personal info');
+    }
+
+    public function rules(): array
+    {
+        return [
+           'phone' => ['required'],
```

Comment:
What type of date should 'phone' have? `string` perhaps?

Reply:
Yes, you are right